src/Standards/Generic/Sniffs/WhiteSpace/LanguageConstructSpacingSniff.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Ensures all language constructs contain a single space between themselves and their content.
+ *
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\Generic\Sniffs\WhiteSpace;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+class LanguageConstructSpacingSniff implements Sniff
+{
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return array(
+                T_ECHO,
+                T_PRINT,
+                T_RETURN,
+                T_INCLUDE,
+                T_INCLUDE_ONCE,
+                T_REQUIRE,
+                T_REQUIRE_ONCE,
+                T_NEW,
+                T_YIELD,
```

Comment:
OK... I will need to check it out if this sniff has the same behaviour with two word tokens.. I am curious about it.